migrations/_alembic/env.py

```diff
@@ -11,7 +12,7 @@
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
-fileConfig(config.config_file_name)
+# fileConfig(config.config_file_name)
```

Comment:
Would it to be better to delete this now (and the import earlier) if it's not going to be used?